### ISSUE TITLE
Comma Separated Numbers In Number Field To Be Used With IN Operator

### ISSUE THREAD

**Issue by JasonGluck** [metabase/metabase#5457]
- Your browser and the version: Chrome 59
- Your operating system: OS X 10.12
- Your databases: Redshift
- Metabase version: 0.24.2
- Metabase hosting environment: Ubuntu
- Metabase internal database: Postgres

Recently upgraded from v0.22.2 to v0.24.2 bug is in 0.24.2. 

Previously user was able to enter a comma separated list of numbers into the ids_list field (variable type of number). Now, when entering a comma separated list of numbers only the first number shows in the results of the query.

 [[id IN ({{ids_list}})]]

e.g. If a user enters 10, 11 in the ids_list field the query will only return result for id 10. In v0.22.2 query would return aggregated results for id 10 and 11.

**Comment by salsakran** [comment]
Elevating priority as it's a regression of a commonly requested feature that we shipped a while ago.

**Comment by JasonGluck** [comment]
@attekei don't believe this is a duplicate of #3510.

**Comment by attekei** [comment]
@JasonGluck Me neither, the way that [GitHub's new keyboard update feature](https://github.com/blog/2398-manage-issues-and-pull-requests-with-keyword-updates) works doesn't currently really make sense.

**Comment by attekei** [comment]
Identified that this is a backend issue with parameter parsing. Testing this query against v0.22.2 and current `master` produces different results:

```
POST /api/dataset
{"database":1,"type":"native","native":{"query":"SELECT * FROM ORDERS [[where id IN ({{ids_list}})]]","template_tags":{"ids_list":{"id":"182c0687-d1de-078b-55ba-54cf584a3c8d","name":"ids_list","display_name":"Ids list","type":"number"}}},"parameters":[{"type":"category","target":["variable",["template-tag","ids_list"]],"value":"1,2,3"}]}
```
Currently the backend seems to strip the commas and returns the row with ID 123 instead of returning rows for IDs 1, 2 and 3. Ping @camsaul and @senior

**Comment by camsaul** [comment]
Fixed by #5548

**Comment by JasonGluck** [comment]
Thanks @camsaul !

@samiyabroudi looks like this fix is going to make it into the 0.25 release
